test/torchaudio_unittest/functional/functional_cpu_test.py

```diff
@@ -121,16 +121,18 @@ def test_DB_to_amplitude(self):
         torch.testing.assert_allclose(x2, spec, atol=5e-5, rtol=1e-5)
 
 
-@pytest.mark.parametrize('complex_tensor', [
-    torch.randn(1, 2, 1025, 400, 2),
-    torch.randn(1025, 400, 2)
-])
-@pytest.mark.parametrize('power', [1, 2, 0.7])
-def test_complex_norm(complex_tensor, power):
-    expected_norm_tensor = complex_tensor.pow(2).sum(-1).pow(power / 2)
-    norm_tensor = F.complex_norm(complex_tensor, power)
-
-    torch.testing.assert_allclose(norm_tensor, expected_norm_tensor, atol=1e-5, rtol=1e-5)
+class TestComplexNorm(common_utils.TorchaudioTestCase):
+
+    @parameterized.expand([
```

Comment:
I suppose this is not quite right. `list(itertools.product(...)` might be the right way to go about this. However that would require importing `import itertools`. 

@mthrok which would be the best way to go about this?.

Reply:
Yeah, that's right. That's the pattern I use everywhere. `itertools` is a standard library so there is no problem.
Maybe it's good time to add a helper decorator for `@partameterized.expand(list(itertools.product` pattern, but let's not bother at this moment.